REFERENCE.md

```diff
@@ -25,7 +25,7 @@
 
 * [`gitlab_ci_runner::register`](#gitlab_ci_runnerregister): A function that registers a Gitlab runner on a Gitlab instance. Be careful, this will be triggered on noop runs as well!
 * [`gitlab_ci_runner::register_to_file`](#gitlab_ci_runnerregister_to_file): A function that registers a Gitlab runner on a Gitlab instance, if it doesn't already exist, _and_ saves the retrieved authentication token t
-* [`gitlab_ci_runner::to_toml`](#gitlab_ci_runnerto_toml): Convert a data structure and output to TOML.
+* [`gitlab_ci_runner::to_toml`](#gitlab_ci_runnerto_toml): }
```

Comment:
This is odd.  What happened to the docs?

Reply:
I ahd the same result.  I tired to downgrade puppet-string without success.

Then I gave git bisect a try:

```sh-session
romain@zappy ~/Projects/voxpupuli/puppet-gitlab_ci_runner % git bisect good 
3e4b6811d09bc1c21ab8eaa9cdb33a7a9babb285 is the first bad commit
```

Indeed, when I run:

```sh-session
romain@zappy ~/Projects/voxpupuli/puppet-gitlab_ci_runner % bundle exec rake reference 
[warn]: Could not find file: DO
[warn]: Could not find file: NOT
[warn]: Could not find file: EDIT
[warn]: Could not find file: #
[warn]: Could not find file: https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
Files:                   14
Modules:                  4 (    4 undocumented)
Classes:                  1 (    1 undocumented)
Constants:                0 (    0 undocumented)
Attributes:               1 (    0 undocumented)
Methods:                  7 (    5 undocumented)
Puppet Classes:           5 (    0 undocumented)
Puppet Data Types:        0 (    0 undocumented)
Puppet Data Type Aliases:     4 (    0 undocumented)
Puppet Defined Types:     1 (    0 undocumented)
Puppet Types:             0 (    0 undocumented)
Puppet Providers:         0 (    0 undocumented)
Puppet Functions:         0 (    0 undocumented)
Puppet Tasks:             2 (    0 undocumented)
Puppet Plans:             0 (    0 undocumented)
 60.00% documented
```

Amusingly, it is not the root cause of the failure.  I openned #123 which fix it on my system, but the weird warning persist, and they seem not to come from code in the module :confused: